### PR TITLE
Helper methods for poolManagerMbean

### LINKED ISSUE #8895: APSZOS-270 - Enhance PoolManagerMBean to expose jndiName, maxSize, size and available
The PoolManagerMBeanImpl currently has a toString() method which exposes several pieces of information that are useful to someon debugging connection management issues.  However it would be more useful if those pieces of information are available individually.  The following methods should be added, and their implementation taken what is currently in the toString() method:

```java
public String getJndiName();
public long  getMaxSize();
public long getSize();
public long getAvailable();
```

### PR DESCRIPTION
**Pull request by AlvinChacko**
fixes #8895

### CODE REVIEW

**Comment by AlvinChacko** [comment]
#build

**Comment by AlvinChacko** [comment]
#build

**Comment by AlvinChacko** [comment]
#build

**Comment by AlvinChacko** [comment]
#build

File: dev/com.ibm.ws.jca.cm/bnd.bnd [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #*******************************************************************************
-# Copyright (c) 2017,2018 IBM Corporation and others.
```
Comment (tevans78) [inline dev/com.ibm.ws.jca.cm/bnd.bnd]: Only specify two dates, the first and last; `2017, 2019`

**Comment by AlvinChacko** [comment]
#build

File: dev/com.ibm.ws.jca.cm/src/com/ibm/ws/jca/cm/mbean/package-info.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2017 IBM Corporation and others.
```
Comment (tevans78) [inline dev/com.ibm.ws.jca.cm/src/com/ibm/ws/jca/cm/mbean/package-info.java]: year of first publication and year of last publication ... so `2017, 2019`

**Comment by AlvinChacko** [comment]
#build

**Comment by AlvinChacko** [comment]
#build

File: dev/com.ibm.ws.jca.cm/src/com/ibm/ejs/j2c/PoolManagerMBeanImpl.java [hunk 3]
```diff
@@ -228,19 +289,15 @@ private String toStringExternal() {
...
         buf.append("maxPoolSize=");
-        buf.append(_pm.maxConnections);
+        buf.append(getMaxSize());
         buf.append(nl);
         buf.append("size=");
-        buf.append(_pm.totalConnectionCount.get());
+        buf.append(getSize());
         buf.append(nl);
-        int numUnsharedConnections = 0, numSharedConnections = 0, numFreeConnections = 0;
+        int numUnsharedConnections = 0, numSharedConnections = 0, numFreeConnections = 0;;
```
Comment (ginnick) [inline dev/com.ibm.ws.jca.cm/src/com/ibm/ejs/j2c/PoolManagerMBeanImpl.java]: Extract semicolon.

File: dev/com.ibm.ws.jca.cm/src/com/ibm/ws/jca/cm/mbean/ConnectionManagerMBean.java [hunk 4]
```diff
@@ -109,4 +109,40 @@
...
+     * Displays the JndiName of the connection pool associated with
+     * this Connection Manager as a human readable string.
+     *
+     * @return A non-localized string displaying the JndiName of the connection pool
+     *
+     */
+    public String getJndiName();
+
+    /**
+     * Displays the maximum size of the connection pool associated with
```
Comment (ginnick) [inline dev/com.ibm.ws.jca.cm/src/com/ibm/ws/jca/cm/mbean/ConnectionManagerMBean.java]: Returning a Long not a String.  Need an update to the Abstract as well as the @return description.  For this method and the following 3.

**Comment by AlvinChacko** [comment]
#build

**Comment by AlvinChacko** [comment]
#run-libby-bot

**Comment by AlvinChacko** [comment]
#build - As the perv one was abandoned

**Comment by AlvinChacko** [comment]
The build has an issue that doesnt seem to be related.

**Pull request merged**